app/services/responsibility_service.rb

```diff
@@ -1,30 +1,92 @@
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/CyclomaticComplexity
 class ResponsibilityService
+  NPS = 'NPS'
+  CRC = 'CRC'
+  PRISON = 'Prison'
+  PROBATION = 'Probation'
+  UNKNOWN = 'Unknown'
+  RESPONSIBLE = 'Responsible'
+  SUPPORTING = 'Supporting'
+
   def self.calculate_case_owner(offender)
     case offender.case_allocation
-    when 'NPS'
+    when NPS
       nps_calculation(offender)
-    when 'CRC'
-      'Prison'
+    when CRC
+      PRISON
     else
-      'Unknown'
+      UNKNOWN
     end
   end
 
   def self.calculate_pom_responsibility(offender)
-    return 'Unknown' if offender.release_date.nil?
-    return 'Responsible' if assign_responsible?(offender)
+    return UNKNOWN if offender.release_date.nil?
```

Comment:
I've introduced guard clauses in order to remove if / else statements.
I'm not sure if iterating over rules will bring about much benefit, if anything might make the card harder to read? Welcome opinions!

Reply:
Also welcome suggestions regarding code climate issues (too may return methods). The class method could be split but then I'm not sure if this then makes the code more readable?

Reply:
Looks already and is pretty clear.  Might be tempted to pre-calculate the values to save calling functions multiple times, and I don't think welsh? is required after the first guard?

Reply:
Thanks for the comments - I've pushed the changes :-)